runtime/compilers/rillv1/parse_connector.go

```diff
@@ -26,5 +28,26 @@ func (p *Parser) parseConnector(node *Node) error {
 
 	r.ConnectorSpec.Driver = tmp.Driver
 	r.ConnectorSpec.Properties = tmp.Defaults
```

Comment:
We should exclude entries that end up in `PropertiesFromVariables` from here

Reply:
The downstream code handles the logic that `PropertiesFromVariables` from `Properties` is a copy of `Properties` whose value is a variable. 
Will change it.